service/storage_service.cc

```diff
@@ -3720,7 +3720,6 @@ future<> storage_service::node_ops_abort(utils::UUID ops_uuid) {
         if (as && !as->abort_requested()) {
             as->request_abort();
         }
-        co_await _repair.local().abort_repair_node_ops(ops_uuid);
```

Comment:
I think this change is incorrect. 

storage_service::node_ops_abort has nothing to do with the tracker::shutdown.

It is used to abort repair started by repair based node ops which can happen without the full scylla process shutdown!

Reply:
But why exactly is it incorrect? I remove repair kick from storage service, because it's not needed, since repair kicks itself on shutdown

Reply:
storage_service::node_ops_abort is called when we want to abort a repair based node operation, for example, we started a replace operation but we want to abort it. At this time, none of the nodes are in the shutdown process. 

We want to abort repair not just in shutdown.